README.md

```diff
@@ -134,7 +134,7 @@ is running. You will still need to start the bundle using the karaf commands men
 
 If you are trying to use the Composer plugin permanently, it likely makes more sense to do the following:
 
-* Copy the bundle into `<nexus_dir>/system/org/sonatype/nexus/plugins/nexus-repository-composer/0.0.2/nexus-repository-composer-0.0.2.jar`
+* Copy the bundle into `<nexus_dir>/system/org/sonatype/nexus/plugins/nexus-repository-composer/0.0.8/nexus-repository-composer-0.0.8.jar`
```

Comment:
Maybe use "pseudo version" or "beverage worthy magic" or no change from what you have. Your call.